src/sentry/conf/server.py

```diff
@@ -976,8 +976,6 @@ def create_partitioned_queues(name):
     "organizations:dashboards-basic": True,
     # Enable custom editable dashboards
     "organizations:dashboards-edit": True,
-    # Enable dashboards manager.
-    "organizations:dashboards-manage": False,
```

Comment:
Was this intentional?

Reply:
yes 😛  references to this flag were removed months ago